crates/sui-framework/tests/safe_tests.move

```diff
@@ -23,22 +23,22 @@ module sui::safe_tests {
         };
     }
 
-    // Delegates the safe to delegatee and return the capability ID.
```

Comment:
Hmm I don't think this file should be changed. Here we actually want to use the word "delegate" because we are delegating the capability to transfer money to someone else.

Reply:
<img src="https://media4.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif"/>

Will fix :)